spark/src/test/scala/ai/chronon/spark/test/TableUtilsFormatTest.scala

```diff
@@ -0,0 +1,192 @@
+package ai.chronon.spark.test
+
+import ai.chronon.api.{DoubleType, IntType, LongType, StringType, StructField, StructType}
+import ai.chronon.spark.SparkSessionBuilder.FormatTestEnvVar
+import ai.chronon.spark.test.TestUtils.makeDf
+import ai.chronon.spark.{IncompatibleSchemaException, SparkSessionBuilder, TableUtils}
+import org.apache.spark.sql.functions.col
+import org.apache.spark.sql.{AnalysisException, DataFrame, Row, SparkSession}
+import org.junit.Assert.{assertEquals, assertTrue}
+import org.junit.Test
+
+import scala.util.Try
+
+class TableUtilsFormatTest {
```

Comment:
@piyush-zlai If this test specific to delta lake?

Is it reasonable to add other formats like Hive or Iceberg to the CI?